core/kazoo_amqp/src/api/kapi_dialplan.erl

```diff
@@ -30,6 +30,7 @@
         ,execute_extension/1, execute_extension_v/1
         ,break/1, break_v/1
         ,play/1, play_v/1, playstop/1, playstop_v/1
+        ,playseek/1, playseek_v/1 %TODO
```

Comment:
No longer todo right? LOL

Reply:
@jamesaimonetti latest code has been updated for this, please take a look :)